PufferyKit/Sources/PufferyUI/MessageCell.swift

```diff
@@ -19,6 +25,13 @@ struct MessageCell: View {
                     .font(.headline)
 
                 SelectableTextView(message.body, font: UIFont.preferredFont(forTextStyle: .subheadline))
+                
+                HStack {
+                    Spacer()
+                    Text(messageDateDescription)
+                        .font(.caption)
+                        .foregroundColor(message.color.secondary)
+                }
```

Comment:
Unsure regarding the extra vertical space we need here. Can you move it at the end of the title line?

```
Title    rel date
Some Message body
```